pyclaragenomics/test/test_cudapoa_bindings.py

```diff
@@ -20,7 +20,8 @@
 def test_cudapoa_simple_batch():
     device = cuda.cuda_get_device()
     free, total = cuda.cuda_get_mem_info(device)
-    batch = CudaPoaBatch(10, 0.9 * free, deivce_id=device)
+    batch = CudaPoaBatch(10, 1024, 0.9 * free, deivce_id=device,
```

Comment:
one last request @rilango  - let's add a version of the `test_cudapoa_simple_batch` that uses `banded=True`

Reply:
Thanks @rilango ! Looks great!